src/scripts/sidebar.js

```diff
@@ -192,12 +192,20 @@ class SideBar extends EventEmitter {
         document.body.classList.add('hide-server-list');
         localStorage.setItem('sidebar-closed', 'true');
         this.emit('hide');
+        if (process.platform === 'darwin') {
+            [].forEach.call(document.getElementsByTagName('webview'),
```

Comment:
The first time I tried I got an error that's why a went getElementsByTagName, but I guess the error was related to something else because it's working now 👍